src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
```

Comment:
Would be better to create an calculator object and return it

Reply:
Actually, I see no point to that action